tsconfig.json

```diff
@@ -7,10 +7,11 @@
     "moduleResolution": "node",
     "noImplicitAny": true,
     "strict": true,
+    "preserveSymlinks": true,
```

Comment:
Ran into this? https://github.com/TypeStrong/ts-node/issues/797#issuecomment-492239440